content/research_lines/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Research Lines
+name: SSE
```

Comment:
Why does the index page of the research lines need a name?

Reply:
This was intended for the sustainable software engineering page and should be resolved now (before previous meeting)